src/screens/GroupSetting/index.js

```diff
@@ -32,14 +33,14 @@ const GroupSetting = ({navigation, route}) => {
     updateName,
     submitData,
     lounchGalery,
-    renderHeaderSubtitleText
+    renderHeaderSubtitleText,
+    loadingUpdate
   } = useGroupSetting({navigation, route});
   const isFocusChatName = route?.params?.focusChatName;
   const [channelState] = React.useContext(Context).channel;
 
   const anonymousName = `Anonymous ${channelState?.data?.anon_user_info_emoji_name}`;
   const getProfileName = (name) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `loadingUpdate` variable is introduced but it's not clear where it's coming from. Make sure it's properly defined and initialized in the `useGroupSetting` hook.
```diff
-    loadingUpdate
+    loadingUpdate = false // or some default value
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->